webanno-api-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/api/annotation/model/AnnotationPreference.java

```diff
@@ -119,21 +128,59 @@ public void setScrollPage(boolean aScrollPage)
         scrollPage = aScrollPage;
     }
 
+    /**
```

Comment:
This type of empty JavaDoc doesn't serve any purpose. Could you please remove it (unless of course you would like to write proper JavaDoc ;) )